google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -734,7 +734,11 @@ def create_file file, path = nil, acl: nil, cache_control: nil,
         # @param [OpenSSL::PKey::RSA, String] private_key Service Account's
         #   Private Key.
         # @param [Hash] query Query string parameters to include in the signed
-        #   URL, such as response-content-disposition.
+        #   URL. The given parameters are not verified by the signature.
+        #   Parameters such as `response-content-disposition` and
+        #   `response-content-type` are only included when the file metadata is
```

Comment:
I think "included" might lead to some confusion. The given parameters will always be applied to the URL. But the behavior of the HTTP response using those parameters will only be applied when the file's metadata is missing those values. Maybe "active" instead?